### ISSUE TITLE
Save and preview button

### ISSUE THREAD

**Issue by arknu** [umbraco/Umbraco-CMS#15360]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

12.3.3

### Bug summary

The Save and preview button in the backend does not show any progress indication while it performing the save operation. This is problematic for large sites, where Saving might take 5 or 10 seconds or even longer. Currently the button just appears hung for 5-10 seconds and then suddenly the preview opens.

### Steps to reproduce

1. Press Save. Note the progress animation while the action is being performed.
2. Press Save and preview. Notice the lack of animation indicating a Save is being performed.

### Expected result / actual result

There should be a clear indication that an action is happening

**Comment by elit0451** [comment]
Hi @arknu 👋 Thanks for reaching out.

I managed to reproduce this by adding some throttling to the requests - here is a video of the current (non-ideal) behavior:

https://github.com/umbraco/Umbraco-CMS/assets/21998037/ad925668-372e-43d6-8de8-baecb743e43d

Users get no feedback when clicking the "Save and preview" button. I will mark this as up for grabs as we would like some help fixing this 🙂
